XenAdmin/Controls/CustomDataGraph/DataSet.cs

```diff
@@ -57,10 +57,7 @@ public class DataSet : IComparable<DataSet>
         private bool _hide;
         private bool _deselected;
 
-        /// <summary>
-        /// A guess at the data type
-        /// </summary>
-        public DataType Type;
+        public DataType Category;

```

Comment:
Just checking that the comment's removal was done on purpose here. Ignore if that's the case.

Reply:
I removed it because it's misleading. We're not guessing the type, just assigning a category so it's easier for the user to find the graphs (the category is shown on the Performance page > Actions button > New Graph dialog).